src/sites/link/fas.li.js

```diff
@@ -48,34 +45,15 @@ _.register({
 });
 
 _.register({
-  rule: [
+  rule: 
     {
       host: /^cpmlink\.net$/,
       path: /^\/go\/[\w-]+$/,
     },
-    {
-      host: /^(www\.)?croco\.(me|site)$/,
-      path: /^\/ok\/\w+$/,
-    },
-  ],
   async ready () {
     let a = $('#btn-main');
     const i = a.href.lastIndexOf('http');
     a = a.href.substr(i);
     await $.openLink(a);
   },
 });
-
-_.register({
-  rule: {
-    host: [
-      /^fas\.li$/,
-      /^(www\.)?shink\.me$/,
```

Comment:
i think we should keep this part.. as  shink.me  site is up

Reply:
Again, shink.me was in separate sections. Although I think I got this one backwards. That whole first section isn't really needed at all.

Reply:
Resolved with c1988ed